modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -342,6 +343,8 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("EmptyAction", "BCs/Pressure");
   syntax.registerActionSyntax("PressureAction", "BCs/Pressure/*");
 
+  syntax.registerActionSyntax("GeneralizedPlaneStrainAction", "GeneralizedPlaneStrain/*");
```

Comment:
:+1: for @permcody's comment. This should be under `[Modules/TensorMechanics/GeneralizedPlaneStrain/*]`. The other actions in this file are all _single_ task actions. So this new one is different. Rather than putting it top level (which should be reserved for _applications_!) I think it should go under the Modules block.